CalendarFXView/src/main/java/impl/com/calendarfx/view/DayEntryViewSkin.java

```diff
@@ -117,13 +147,16 @@ private Label createTitleLabel() {
         return label;
     }
 
-    private void updateLabels() {
+    /**
+     * This method will be called if the labels needs to be updated
```

Comment:
Grammar: "needs" -> "need". Also add a full stop at the end of your javadocs please.